### DISCUSSION TITLE
Is mLauncher project closed? the repo is archived.

### DISCUSSION THREAD

**Discussion by WiseSmil3** [DroidWorksStudio/EasyLauncher#182] [General]
### Select Topic Area

Question

### Body

I have been a long long time mLauncher user, I love almost everything about it. But in the recent update (1.6.9) the percentage icon has changed and moved to the upper right corner. I personally put text, lock, and calendar all centered the new clock position brakes the style. 

I want to thank #HeCodes2Much for keeping the mLauncher maintained. I have many friends who love mLauncher as well.

I can see that easyLauncher is trying to do the same, if not replace mLauncher. DroidWorks Studio please at least keep the costumazations options from mLauncher into easyLauncher.

Again Thank you HeCodes2Much and DroidWorks Studio for your hard work.

Your unknown long term supporter.

**Comment by CreativeCodeCat** [comment]
Yes I am stopping work on mLauncher to focus more on EasyLaucher.

If you could give me a list of things you would like to see added from mLauncher to EasyLauncher I would be happy to port things over.

**Reply by WiseSmil3** [reply]
Futures migration:

Have the option to display the Work profile suitcase icon to the apps in app drawer.

Do not display keyboard in the app list option

The animations from mLauncher are much better and more smooth.

Make it so I can use the gestures on all scrollable surfaces not just the empty spaces.



Fix:

The animations when opening the app drawer to mach the gesture direction.

Make the top and bottom of the app drawer fade as scrolling rather then abrupt. 

Then changing the home screen text sizes and arrangements have a view port to display the changes.

Then reaching the bottom of the home page enter app drawer. 


Gugs:

The UX not responding correctly when interacting with work profile apps in the app drawer.

App drawer apps not displaying correctly when interacting with work profile apps

**Reply by CreativeCodeCat** [reply]
I don't really care for work profile so I don't care to add support for them.

**Reply by WiseSmil3** [reply]
For the privacy oriented users, work profile is used for app isolation. This way one phone can have two of everything. Like having two phones at ones. Coupled with GrapheneOS puts your data in your control.

I love that the mLauncher does not require any promitions. Hope this carries on to easyLauncher.

**Comment by Zumux** [comment]
May I suggest that you mention somewhere in the readme for mlauncher about this?

**Reply by CreativeCodeCat** [reply]
Can't make edits once it is archived

**Reply by dallyger** [reply]
Got confused too. stumbled upon the archived repository without any notice on why. Took a while to find this project and this discussion thread. Thank you for maintaining mLauncher until now! Will look forward to this new launcher and try it in the following days.
